profile/README.md

```diff
@@ -1,4 +1,10 @@
-<p align="left"><img src="https://github.com/user-attachments/assets/b31e430e-0ab6-42e3-ba43-d96f5aa61d94" width="500" /></p>
+<p align="left">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://github.com/user-attachments/assets/b31e430e-0ab6-42e3-ba43-d96f5aa61d94">
```

Comment:
nit: I assume we're adding this to be exhaustive, but it appears unnecessary given it's already in the default img, right?

Reply:
Yeah. I'll remove.